README.md

```diff
@@ -9,6 +9,14 @@ with [Rocket](https://rocket.rs/).
 _ferrocarril_ means _railway_ in Spanish and sounds like _ferrous_ which means
 _containing iron_.
 
+## Usage
```

Comment:
foolsgold is one of several binary crates in this workspace. It is mostly a proof of concept/demo. Can we scope this usage under the `## foolsgold` header as an h3?